guides/v2.2/cloud/project/project-upgrade.md

```diff
@@ -45,126 +39,145 @@ To create a temporary `config.php` file:
+
+### Back up the database
 
 {% include cloud/backup-db.md %}
 
-## Complete the upgrade
+### Complete the upgrade
 
-1. Change to your Magento root directory and set the upgrade version.
+If you use PHP version 7.2, you must remove the `mcrypt` extension from the [`extensions` section of the `.magento.app.yaml` file]({{page.baseurl}}/cloud/project/project-conf-files_magento-app.html#configure-php-options). For Pro projects, you need to create a support ticket to completely disable the `mcrypt` extension.
 
-    ```bash
-    composer require magento/magento-cloud-metapackage <requiredversion> --no-update
-    ```
+{:.procedure}
+To upgrade the Magento version:
+
+1. When upgrading to 2.3, you must [update the `autoload`](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html#update-autoload) property of the `composer.json` file.
+
+1. Change to your Magento root directory and set the upgrade version using the [version constraint syntax]({{page.baseurl}}/cloud/project/ece-tools-upgrade-project.html#metapackage).
+
+   ```bash
+   composer require "magento/magento-cloud-metapackage":">=2.x.2 <2.x.3" --no-update
+   ```
+
+   {: .bs-callout-info }
+   You must use the version constraint syntax to successfully update the `{{site.data.var.ct}}` package.
 
 1. Update the project.
 
-    ```bash
-    composer update
-    ```
+   ```bash
+   composer update
+   ```
 
 1. Add, commit, and push code changes.
 
-    ```bash
-    git add -A && git commit -m "Upgrade" && git push origin <branch name>
-    ```
+   ```bash
+   git add -A && git commit -m "Upgrade" && git push origin <branch-name>
+   ```
 
-    `git add -A` is required to add all changed files to source control because of the way Composer marshals base packages. Both `composer install` and `composer update` marshal files from the base package (`magento/magento2-base` and `magento/magento2-ee-base`) into the package root.
+   `git add -A` is required to add all changed files to source control because of the way Composer marshals base packages. Both `composer install` and `composer update` marshal files from the base package (`magento/magento2-base` and `magento/magento2-ee-base`) into the package root.
 
-    The files Composer marshals belong to the new version of Magento, to overwrite the outdated version of those same files. Currently, marshaling is disabled in Magento Commerce, so you must add the marshaled files to source control.
+   The files that Composer marshals belong to the new version of Magento, to overwrite the outdated version of those same files. Currently, marshaling is disabled in {{site.data.var.ee}}, so you must add the marshaled files to source control.
 
 1. Wait for deployment to complete.
 
 1. Verify the upgrade in your Integration, Staging, or Production environment by using SSH to log in and check the version.
 
-    ```bash
-    php bin/magento --version
-    ```
+   ```bash
+   bin/magento --version
```

Comment:
Is there any standard on running binary files? I see our hooks using PHP binary `php ...` and this line does not have it

Reply:
Yes, they removed PHP from a lot of the core documentation, but I haven't always allowed that to happen in the Cloud documentation. This one slipped by me. If you think it is best, I will revert to use PHP.

Reply:
Just asking, don't see a big issue here